test/postman/dbp-test-collection.json

```diff
@@ -900,5 +873,91 @@
 				]
 			}
 		}
+	],
+	"variable": [
+		{
+			"id": "3de1ca96-1c37-424d-9397-d7edf8a68a3b",
+			"key": "acsUrl",
+			"value": "{{url}}/alfresco",
+			"type": "string",
+			"description": ""
+		},
+		{
+			"id": "41445b16-4bba-4d2c-afb5-5451a75988ca",
+			"key": "shareUrl",
+			"value": "{{url}}/share",
+			"type": "string",
+			"description": ""
+		},
+		{
+			"id": "f56e62cc-25ac-44f1-8208-e6220ad69e18",
+			"key": "apsUrl",
+			"value": "{{url}}/activiti-app",
+			"type": "string",
+			"description": ""
+		},
+		{
+			"id": "aaf6ca65-cd8a-44be-881f-b5fa35fdf4d0",
+			"key": "identityServiceUrl",
+			"value": "{{url}}/auth",
+			"type": "string",
+			"description": ""
+		},
+		{
+			"id": "e32b0a99-5c6a-40b4-9e6b-ba95340ceab1",
+			"key": "identityServiceRealm",
+			"value": "springboot",
```

Comment:
By default shouldn't be alfresco?

Reply:
Thanks for catching this, I've pushed a new commit that addresses this.